pkg/tools/tools.go

```diff
@@ -72,23 +72,34 @@ func SocketCleanup(listenEndpoint string) error {
 	return nil
 }
 
+// Unix socket file path.
```

Comment:
Can you explain why you need that?

Reply:
I replaced unix socket path with more general `net.Addr` to be able to connect to Dataplane from outside of cluster and program it directly. So instead of hard-coded `"unix"` socket type some of the API functions now accept `net.Addr`.  `SocketPath` is a simple adapter type that allows casting path strings to `net.Addr` where needed.